sites/docs/pages/reference/themes/index.md

```diff
@@ -0,0 +1,526 @@
+---
+sidebar_position: 4
+hide_table_of_contents: false
+title: Themes
+description: Customize the appearance of your Evidence application in light and dark mode using our theming system.
+sidebar_badge: New!
+---
+
+Customize the appearance of your Evidence application in light and dark mode using our theming system.
+
+# Appearance
```

Comment:
Should this section be called dark mode, or appearance mode?

Reply:
Changed to "Appearance Modes" here [c59c9a8](https://github.com/evidence-dev/evidence/pull/2871/commits/c59c9a8c249059e069aa0f3eb099bcc4068d47f2)